odsApi_versioned_docs/version-7.3/whats-new/whats-new-in-this-release.md

```diff
@@ -19,28 +19,64 @@ Ed-Fi ODS / API Platform v7.3 and provides links to additional documentation.
 
-### Permissions API
+Ed-Fi ODS/API v7.3 introduces partitioned cursor-based paging to significantly
+reduce query overhead when paging through large datasets. This approach provides
+an alternative to traditional offset-based pagination. It uses lightweight,
+sequential page tokens while retaining parallel processing capabilities through
+the partitions endpoint. For instructions on using this feature, refer to
+[Improve Paging Performance with Partitioned Cursor
+Paging](./../client-developers-guide/improve-paging-performance-cursor-paging.md).
+To learn more about the design and the performance improvements it offers, refer
+to [Cursor Paging for Improved Data Out
+Performance](./../technical-articles/cursor-paging-implementation-for-improved-data-out-performance.md)
+
+### Search by Identification Code
+
+This release introduces the ability to query resources by identification codes,
+enabling just-in-time lookups instead of requiring maintenance of a local
+mapping of identification codes used within the ecosystem. This enhancement
+simplifies API integration for clients rostered with identification codes that
+differ from identifiers used by the API host.
+
+### Token Info and Permissions
+
+Ed-Fi ODS/API v7.3 enhances the /oauth/token_info endpoint to include
+resource-level permissions, along with namespace prefixes, associated education
+organizations, and profiles assigned to a token. Refer to [Token
+Info](./../client-developers-guide/authorization.md/#token-info) for details.
 
 ### Extensible Authorization Filtering
 
+Ed-Fi ODS/API v7.3 introduces powerful and flexible mechanism to define
+authorization strategies without requiring code changes. This feature allows
+implementers to create database views tailored to specific authorization needs,
+such as restricting access by student program enrollment or grade level.
+Granular security setup can be achieved simply by creating a custom database
+view and configuring the necessary metadata without requiring recompilation or
+API process restart. This enhancement streamlines the implementation of nuanced
+security strategies, enabling greater control and adaptability for diverse use
+cases. Refer to articles on [Authorizing Requests Using Custom Database
+Views](./../technical-articles/authorizing-requests-using-custom-database-views.md)
+and [How To: Use Custom View-Based
+Authorization](./../how-to-guides/how-to-use-custom-view-based-authorization.md)
+for details.
+
 ### MetaEd IDE v4.5
 
 Implementing extensions in Ed-Fi ODS / API v7.3 requires implementers to update
-to [MetaEd IDE
-v4.5](https://edfi.atlassian.net/wiki/display/METAED20/MetaEd+Home). Refer to
-[MetaEd 4.5 What's
-New](https://edfi.atlassian.net/wiki/spaces/METAED20/pages/23711107/What%27s+New)
-for details on latest updates and improvements.
+to [MetaEd IDE v4.5](/reference/metaed). Refer to [MetaEd 4.5 What's
+New](/reference/metaed/releases/4.4.0) for details on latest updates and
```

Comment:
Oops, I have forgotten about creating the metaed 4.5 release notes page.

Reply:
I can address updating this with rest of the post release changes(release tags and version link updates) when 4.5 is ready

Reply:
MetaEd 4.5 release notes are now approved in draft, ready to be published once MetaEd is published.